server/channelserver/handlers_quest.go

```diff
@@ -195,23 +196,50 @@ func handleMsgMhfEnumerateQuest(s *Session, p mhfpacket.MHFPacket) {
 	bf := byteframe.NewByteFrame()
 	bf.WriteUint16(0)
 
-	rows, _ := s.server.db.Query("SELECT id, COALESCE(max_players, 4) AS max_players, quest_type, quest_id, COALESCE(mark, 0) AS mark FROM event_quests ORDER BY quest_id")
+	currentTime := time.Now()
+	var tableName = "event_quests"
```

Comment:
Is the data structure for event_quests between versions different enough to require entirely different tables?